# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/smartmontools/noinst-init.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	Sat Apr 19 12:28:47 2003
+++ ./Makefile	Sat Apr 19 12:28:59 2003
@@ -103,7 +103,6 @@
 	rm -f $(DESTDIR)/usr/share/man/man8/smartd.8
 	install -m 755 -D smartctl         $(DESTDIR)/usr/sbin/smartctl
 	install -m 755 -D smartd           $(DESTDIR)/usr/sbin/smartd
-	install -m 755 -D smartd.initd     $(DESTDIR)/etc/rc.d/init.d/smartd
 	install -m 644 -D smartctl.8.gz    $(DESTDIR)/usr/share/man/man8/smartctl.8.gz
 	install -m 644 -D smartd.8.gz      $(DESTDIR)/usr/share/man/man8/smartd.8.gz
 	install -m 644 -D smartd.conf.5.gz $(DESTDIR)/usr/share/man/man5/smartd.conf.5.gz
